smart_campus/app/views.py

```diff
@@ -38,6 +40,16 @@
 )
 
 
+def validate_administrator(function=None):
```

Comment:
Is there any reason that you prefer `validate_administrator` to `administrator_required`?

Reply:
No, I will rename it.